tasks/src/Reinforcement-learning/about.md

```diff
@@ -0,0 +1,9 @@
+## Use Cases
+
+### Robotics
+
+Reinforcement learning can be used to teach robots how to perform tasks by providing them with feedback about their performance. For example, a robot might be given a score for each task it completes, and the robot can use reinforcement learning to learn how to improve its performance by trial and error.
+
+### Gaming
+
+Reinforcement learning is used in gaming to help agents learn how to optimally play a game. This is done by providing the agent with feedback in the form of rewards or punishments after each action is taken. Over time, the agent will learn which actions lead to the highest rewards and will adjust its behaviour accordingly.
```

Comment:
Can you elaborate a little more and avoid using passive voice and maybe explain what an agent is? 🙂